spec/models/form_profile_spec.rb

```diff
@@ -842,6 +842,39 @@
     }
   end
 
+  let(:vform_mock_ae_design_patterns_expected) do
+    {
+      'data' => {
+        'attributes' => {
+          'veteran' => {
+            'firstName' => user.first_name&.capitalize,
+            'middleName' => user.middle_name&.capitalize,
+            'lastName' => user.last_name&.capitalize,
+            'suffix' => user.suffix,
+            'dateOfBirth' => user.birth_date,
+            'ssn' => user.ssn.last(4),
+            'gender' => user.gender,
+            'address' => {
+              'addressLine1' => street_check[:street],
+              'addressLine2' => street_check[:street2],
```

Comment:
Is  addressLine3: [street3] not eede?

Reply:
@RachalCassity  no we don't need addressLine3 right now because this is just for user research.